eahub/templates/emails/message_profile.html

```diff
@@ -0,0 +1,9 @@
+<p>
```

Comment:
Am I right that we're currently not using this, but instead the txt file? You're probably in the middle of this.

I'd suggest that for now, we make "Report an issue" an mailto: link to admin@eahub.org. We can implement a form at a later point. Issue: https://github.com/rtcharity/eahub.org/issues/1139